spec.bs

```diff
@@ -979,15 +979,34 @@ The Shared Storage API will integrate into the [=Storage Model|Storage API=] as
     1. Otherwise, return false.
   </div>
 
+
+Extension to the {{Window}} interface {#window-extension}
+=====================================================
+Each {{Window}} object has an associated {{SharedStorage}} instance {{Window/sharedStorage}}, which is created alongside the .{{Window}} if [=Shared Storage=] is enabled, with the [=Window/sharedStorage getter|getter=] below.
```

Comment:
s/alongside the .{{Window}}/alongside the {{Window}}/

Reply:
Done.